<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameStartDialog</class>
 <widget class="QDialog" name="GameStartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>対局情報</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>先手</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>手合割:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>平手</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>香落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>角落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>飛車落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>飛香落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>二枚落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>四枚落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>六枚落ち</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="sentePlayerName">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>名前</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>種類:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="senteHumanType">
        <item>
         <property name="text">
          <string>人</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CPU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>後手</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>手合割:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>平手</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>香落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>角落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>飛車落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>飛香落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>二枚落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>四枚落ち</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>六枚落ち</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="gotePlayerName">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>名前</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="goteHumanType">
        <item>
         <property name="text">
          <string>人</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CPU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TCP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>種類:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>対局情報</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>持ち時間:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="limitTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>30</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GameStartDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GameStartDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
